README.md

```diff
@@ -95,6 +95,6 @@ repeat these steps to upgrade to future releases.
 ## Building
 
 * [Linux](docs/build-instructions/linux.md)
-* [macOS](docs/build-instructions/os-x.md)
+* [macOS](docs/build-instructions/mac-os.md)
```

Comment:
Extreme nitpicking, but with the current naming scheme (look at FreeBSD), the file should be named `macos.md`.

---
_PS. Not deliberately trying to :trollface: you by the way, hehe._